languages/scala/Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM debian:buster
+FROM openjdk:11
 
 WORKDIR /tmp
 
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y default-jre \
-    apt-get install -y scala
+RUN \
+    curl -fsL https://downloads.typesafe.com/scala/2.13.6/scala-2.13.6.tgz | tar xfz - -C /usr/share && \
+    mv /usr/share/scala-2.13.6 /usr/share/scala && \
+    chown -R root:root /usr/share/scala && \
+    chmod -R 755 /usr/share/scala && \
+    ln -s /usr/share/scala/bin/* /usr/local/bin
```

Comment:
This isn't how `ln -s` works. It takes arguments in the format `ln -s target destination_1 destination_2 ...` such that all destinations point to the target, so this will replace `/usr/local/bin` and most of the files in `/usr/share/scala/bin/` with links to the one file that gets selected first